### PR TITLE
AST-1540 - var() colors code appearing in block editor theme palette picker.

### PR DESCRIPTION
**Pull request by imnavanath**
### Description
- Due to theme.json var() colors appearing as it is in block editor theme control

### Screenshots
- Issue - https://d.pr/i/TTAmyJ
- With fix - https://share.bsf.io/wbuYbPxq (Little jerk is there, but its on 1ms time. So IMO its okay instead of completing removing the control)

### Types of changes
- Bug fix 

### How has this been tested?
- Test on any new post/page editor's any color picker
- Test environment - WP-5.9

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Comment by premanshup** [comment]
@imnavanath Please get this synced.

File: inc/assets/js/block-editor-script.js [hunk 1]
```diff
@@ -53,6 +53,18 @@ function astra_onload_function() {
 
 	wp.data.subscribe(function () {
 		setTimeout( function () {
+			/**
+			 * In WP-5.9 block editor comes up with color palette showing color-code canvas, but with theme var() CSS its appearing directly as it is. So updated them on wp.data event.
+			 */
+			var customColorPickerButtons = document.querySelectorAll( '.components-color-palette__custom-color' );
+
+			for ( var item = 0;  item < customColorPickerButtons.length; item++ ) {
+				var colorCode = customColorPickerButtons[item].innerText;
```
Comment (premanshup) [inline inc/assets/js/block-editor-script.js:62]:
```suggestion
			const customColorPickerButtons = document.querySelectorAll( '.components-color-palette__custom-color' );

			for ( const btnCount = 0;  btnCount < customColorPickerButtons.length; btnCount++ ) {
				const colorCode = customColorPickerButtons[btnCount].innerText;
```

**Review by premanshup — CHANGES_REQUESTED** [review]

File: inc/assets/js/block-editor-script.js (hunk 1 above)
Comment (imnavanath) [reply inc/assets/js/block-editor-script.js]: Updated code just apart from btnCount, as its getting error to not update const value as its in incrementing loop.
Commit - https://github.com/brainstormforce/astra/pull/4078/commits/61274771daac61764c44d8e92f68751b60008be6

File: inc/assets/js/block-editor-script.js [hunk 2]
```diff
@@ -53,6 +53,18 @@ function astra_onload_function() {
 
 	wp.data.subscribe(function () {
 		setTimeout( function () {
+			/**
+			 * In WP-5.9 block editor comes up with color palette showing color-code canvas, but with theme var() CSS its appearing directly as it is. So updated them on wp.data event.
+			 */
+			const customColorPickerButtons = document.querySelectorAll( '.components-color-palette__custom-color' );
+
+			for ( var btnCount = 0; btnCount < customColorPickerButtons.length; btnCount++ ) {
```
Comment (premanshup) [inline inc/assets/js/block-editor-script.js:61]:
```suggestion
			for ( let btnCount = 0; btnCount < customColorPickerButtons.length; btnCount++ ) {
```

**Review by premanshup — APPROVED** [review]

**Pull request merged**